src/unity/schemas/Accounts.yml

```diff
@@ -1,3 +1,3 @@
 type: array
 items:
-  $ref: './Account.yml'
+  $ref: './UserAccount.yml'
```

Comment:
I'm thinking we should also rename `Accounts` -> `UserAccounts` as you suggested in the PR description. I think that would be less confusing.